install_config/install/docker_registry.adoc

```diff
@@ -1275,6 +1328,33 @@ To prevent this, avoid the tagging of internally managed images completely, or
 re-push the built image to the desired namespace
 link:#access-pushing-and-pulling-images[manually].
 
+[[known-issue-s3-image-push-fails]]
+==== Image Push fails with `500 Internal Server Error` on S3 storage
+
+There are problems reported happening when the registry runs on S3 storage
```

Comment:
Consider rewording:
When the registry runs on the S3 storage back-end, there are reported issues.